projects/schematics/src/migrations/5_0/constructor-deprecations/data/constructor-deprecations.migration.ts

```diff
@@ -0,0 +1,4762 @@
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'CheckoutStepService',
+        importPath: '@spartacus/checkout/base/components',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutLoginComponent',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'FormBuilder',
+        importPath: '@angular/forms',
+      },
+      {
+        className: 'AuthRedirectService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'FormBuilder',
+        importPath: '@angular/forms',
+      },
+      {
+        className: 'AuthRedirectService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'FormBuilder',
+        importPath: '@angular/forms',
+      },
+      {
+        className: 'AuthRedirectService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutOrderSummaryComponent',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutProgressMobileTopComponent',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutStepService',
+        importPath: '@spartacus/checkout/components',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'ActiveCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CheckoutStepService',
+        importPath: '@spartacus/checkout/components',
+      },
+    ],
+    addParams: [
+      {
+        className: 'ActiveCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+      {
+        className: 'CheckoutStepService',
+        importPath: '@spartacus/checkout/base/components',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutReplenishmentFormService',
+    importPath: '@spartacus/checkout/components',
+    deprecatedParams: [],
+    removeParams: [],
+    addParams: [
+      {
+        className: 'EventService',
+        importPath: '@spartacus/core',
+      },
+    ],
+  },
+  {
+    class: 'CheckoutStepsSetGuard',
```

Comment:
only caught checkout that does not make sense since this is the one that split with base / b2b

Reply:
Yup.  We deal with this in milestone 2.